### ISSUE TITLE
(Request) Scientific equipment.

### ISSUE THREAD

**Issue by pooing1234** [Electrical-Age/ElectricalAge#732]
Scientific equipment for developing potions, heaths kits and more.

**Comment by Tesla010** [comment]
Em, this is an electricity mod. Which is so freaking cool.

**Comment by pooing1234** [comment]
@Nicecool19 Yes, I know. But it would be nice to have more things to power. :)

**Comment by Baughn** [comment]
Not 'potions', though. Minecraft already has those.

I'm inclined to accept any request for useful items, if you have more specific ideas as to how they should work.

**Comment by AMIDIBOSS** [comment]
Potion stand heater?

**Comment by Baughn** [comment]
The potion stand... doesn't need heating, though?

**Comment by AMIDIBOSS** [comment]
It does require blaze powder

**Comment by Baughn** [comment]
I *guess*...

I don't think anything that blatantly magical would really fit with the mod, but maybe some old electrical psuedoscience could take its role. Electrocution as a way of gaining temporary potion effects, for example.
